impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -109,8 +114,7 @@ public EndpointPair getControllerEndpoint(int brokerId) {
 
             servers = clusterConfig.getBrokerConfigs(kafkaEndpointsSupplier, zookeeperEndpointSupplier).map(this::buildKafkaServer).collect(Collectors.toList());
 
-        }
-        catch (IOException e) {
+        } catch (IOException e) {
```

Comment:
Must update my formatter settings to match the maven plug-in (thought I don't like the line break it adds)